.github/workflows/release.yaml

```diff
@@ -119,6 +119,9 @@ jobs:
           AC_APIKEY_ISSUER_ID: ${{ secrets.AC_APIKEY_ISSUER_ID }}
           AC_APIKEY_ID: ${{ secrets.AC_APIKEY_ID }}
           AC_APIKEY_FILE: /tmp/apple_apikey.p8
+          CODER_SIGN_WINDOWS: "0"
+          AUTHENTICODE_CERTIFICATE_FILE: /tmp/windows_cert.pkcs12
```

Comment:
Code signing certificates can be obtained via https://sectigo.com/ssl-certificates-tls/code-signing and will require organization validation (ie. a DUNS number and corresponding phone number that works)